src/metric/definition.go

```diff
@@ -565,6 +565,32 @@ var EtcdQueries = []prometheus.Query{
 
 // KSMSpecs are the metric specifications we want to collect from KSM.
 var KSMSpecs = definition.SpecGroups{
+	"job_name": {
+		IDGenerator:     prometheus.FromLabelValueEntityIDGenerator("kube_job_created", "job_name"),
+		TypeGenerator:   prometheus.FromLabelValueEntityTypeGeneratorWithCustomGroup("kube_job_created", "job"),
+		NamespaceGetter: prometheus.FromLabelGetNamespace,
+		MsTypeGuesser:   metricSetTypeGuesserWithCustomGroup("job"),
+		Specs: []definition.Spec{
+			{Name: "createdAt", ValueFunc: prometheus.FromValue("kube_job_created"), Type: sdkMetric.GAUGE},
```

Comment:
The KSM description of each `kube_job_*` metric is listed in the PR description.